lib/command/obelisk-command.cabal

```diff
@@ -24,7 +24,7 @@ library
     , git
     , github
     , here
-    , hnix
+    , hnix >=0.5
```

Comment:
I can't see anything else in the `command` diff related to this. 
For future reference, was this already de-facto or needed for some other reason?